src/Microsoft.AspNetCore.Blazor.Server/Circuits/RenderBatchFormatterResolver.cs

```diff
@@ -4,8 +4,10 @@
 using MessagePack;
 using MessagePack.Formatters;
 using Microsoft.AspNetCore.Blazor.Rendering;
+using Microsoft.IO;
```

Comment:
Asking just in in case: this needs approval approval from @Eilon  - even if its OSS and from Microsoft.